azure-pipelines.yml

```diff
@@ -122,6 +71,13 @@ jobs:
         env:
           DANGER_GITHUB_API_TOKEN: $(DANGER_GITHUB_API_TOKEN)
 
+      - script: |
+          yarn lage build --since $(target_branch) --scope @fluentui/* --no-deps --verbose --no-cache
```

Comment:
This seems mostly redundant with the main build/test/lint step--is all of that really needed here?